kie-server-parent/kie-server-router/kie-server-router-proxy/src/main/java/org/kie/server/router/identity/IdentityService.java

```diff
@@ -0,0 +1,18 @@
+package org.kie.server.router.identity;
+
+import io.undertow.security.idm.IdentityManager;
+
+public interface IdentityService extends IdentityManager {
+
+    default String id() {
+        return "default";
```

Comment:
I think that the default value shouldn't be defined as user is required to specify his service id, otherwise the default value would be used and it wouldn't be clear what implementation is applied (customer one or default one).

Reply:
ok.